translation.rst

```diff
@@ -353,7 +353,7 @@ with these tasks:
 
 .. code-block:: terminal
 
-    # updates the French translation file with the missing strings found in app/Resources/ templates
+    # updates the French translation file with the missing strings found in src/Resources/views/ templates
```

Comment:
Now that this path is deprecated too, we should use `templates/` instead and maybe add the deprecation note for `src/Resources/translations` and `src/Resources/views` directories?